force-app/main/default/flows/add_email_if_its_less_than_20.flow-meta.xml

```diff
@@ -0,0 +1,71 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<Flow xmlns="http://soap.sforce.com/2006/04/metadata">
+    <apiVersion>57.0</apiVersion>
+    <environments>Default</environments>
+    <formulas>
```

Comment:
No need to use resource, Use same formula in decision element when you trying to compare with len 20

Reply:
i am not  using decision element thats why i am use resoucre